x/cvm/client/rest/query.go

```diff
@@ -128,3 +133,31 @@ func metaHandler(cliCtx context.CLIContext) http.HandlerFunc {
 		rest.PostProcessResponse(w, cliCtx, res)
 	}
 }
+
+func contractHandler(cliCtx context.CLIContext) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		bech32addr := vars["address"]
+
+		cliCtx, ok := rest.ParseQueryHeightOrReturnBadRequest(w, cliCtx, r)
+		if !ok {
+			return
+		}
+
+		var baseAcc *auth_types.BaseAccount
```

Comment:
We probably have to use Account interface here to accommodate vesting accounts.

Reply:
Right we need to use authexported.Account or something like that

Reply:
OK, I will check this and try to refine this point

Reply:
thank you!